### PR TITLE
Mandrewdarts/close on country select

### PR DESCRIPTION
**Pull request by Mandrewdarts**
Now the select closes once a country is selected
https://tns.atlassian.net/browse/TRACKENG-8834

https://user-images.githubusercontent.com/7798234/194653098-98fd996a-3867-4834-a0ae-51b55a3fe107.mov

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,14 +20,14 @@
...
 	"dependencies": {
-		"@angular/animations": "~11.2.1",
-		"@angular/common": "~11.2.1",
-		"@angular/compiler": "~11.2.1",
-		"@angular/core": "~11.2.1",
-		"@angular/forms": "~11.2.1",
-		"@angular/platform-browser": "~11.2.1",
-		"@angular/platform-browser-dynamic": "~11.2.1",
-		"@angular/router": "~11.2.1",
+		"@angular/animations": "^13.0.0",
```
Comment (aaronbachman) [inline package.json]: This should be updated to use: `12.2.12` to match

**Review by aaronbachman — CHANGES_REQUESTED** [review]
Summary of conversation:
- [x] Remove the unnecessary changes
- [x] Change it to angular `12.2.12`
- [ ] Run the tests locally and screenshot of them passing
- [ ] If possible add the action to auto run the tests in this repo.

**Comment by aaronbachman** [comment]
Once the changes above are done we should be good to merge this into master which will kick off an action to publish the new package version.  Once that's done the other PR will have to update the version in package.json to `3.1.2`


https://github.com/travelnetsolutions/track-core/pull/17417#issuecomment-1284223830

File: package.json (hunk 1 above)
Comment (Mandrewdarts) [reply package.json]: Updated 👍

**Comment by Mandrewdarts** [comment]
@aaronbachman So the tests for the base library don't work. It looks like they never wrote or ran any tests.

**Review by aaronbachman — APPROVED** [review]
Approved

**Pull request merged**